### ISSUE TITLE
Comment from Grace Hadley:  Accessibility and our facilities:  I don't think these should be bunched together.  Facilities could be put under opening times and then the Access page could be put as the first "read more" section.

### ISSUE THREAD

**Issue by portableant** [FitzwilliamMuseum/beta.fitz.ms#37]
[https://www.bugherd.com/projects/197411/tasks/95](https://www.bugherd.com/projects/197411/tasks/95)

**Comment by portableant** [comment]
@[Daniel Pett](297533) Agreed. Let's discuss
